src/selectedtests/work_items/cli.py

```diff
@@ -7,49 +7,116 @@
     process_queued_test_mapping_work_items,
     process_queued_task_mapping_work_items,
 )
-from selectedtests.helpers import get_evg_api, get_mongo_wrapper, setup_logging
+from selectedtests.datasource.mongo_wrapper import MongoWrapper
+from selectedtests.helpers import get_evg_api, setup_logging
+from selectedtests.evergreen_helper import get_evg_project
+from selectedtests.work_items.project_task_mapping_work_item import ProjectTaskMappingWorkItem
+from selectedtests.work_items.project_test_mapping_work_item import ProjectTestMappingWorkItem
+
+DEFAULT_WEEKS_BACK = 24
 
 
 @click.group()
 @click.option("--verbose", is_flag=True, default=False, help="Enable verbose logging.")
+@click.option("--mongo-uri", required=True, type=str, help="Mongo URI to connect to.")
 @click.pass_context
-def cli(ctx, verbose: str):
+def cli(ctx, verbose: str, mongo_uri: str):
     """Entry point for the cli interface. It sets up the evg api instance and logging."""
     ctx.ensure_object(dict)
-    ctx.obj["mongo"] = get_mongo_wrapper()
+    ctx.obj["mongo"] = MongoWrapper.connect(mongo_uri)
     ctx.obj["evg_api"] = get_evg_api()
 
     setup_logging(verbose)
 
 
 @cli.command()
+@click.option("--project", type=str, required=True, help="Evergreen project to add.")
+@click.option(
+    "--src-regex", type=str, required=True, help="Regular expression for project source files."
+)
+@click.pass_context
+def create_test_mapping(ctx, project: str, src_regex: str):
```

Comment:
Could we rename this to `create_test_mapping_work_item`? At first I thought it did the same thing as the `task-mappings create` command.

Reply:
Because these functions get translated by click to be subcommand names, and their parent command is `work_items` adding work_items again is starting to make the command really long. I would prefer to rename the file `work_items_cli.py` to make it clear these are for work_items.

`work-items create-test-mappings` vs `work-items create-test-mapping-work-items`

Reply:
Ah, okay. That file rename makes sense. 👍

Reply:
Ah, okay. That file rename makes sense. 👍